sdk/src/compute_budget.rs

```diff
@@ -35,8 +35,8 @@ pub enum ComputeBudgetInstruction {
     /// Request a specific maximum number of compute units the transaction is
     /// allowed to consume and an additional fee to pay.
     RequestUnits(u32),
-    /// Additional fee in lamports to charge the payer, used for transaction
-    /// prioritization
+    /// Additional fee in "lamports per 10K CUs" to charge the payer, used for
```

Comment:
I think we can set a higher fee rate unit here (1M instead of 10k) because it gets us more granularity. It means the max fee rate is still u64::MAX (~2*10^19) / (10^6 CU) / (10^9 lamports per SOL) = ~20,000 SOL / CU which should be more than enough I hope 😅 

```suggestion
    /// Set a prioritization fee rate in "lamports per 1M CUs" to charge the payer, used for
```